go/bundle/cmd/gen-bundle/main.go

```diff
@@ -6,11 +6,23 @@ import (
 	"log"
 	"net/url"
 	"os"
+	"strings"
 
 	"github.com/WICG/webpackage/go/bundle"
 	"github.com/WICG/webpackage/go/bundle/version"
 )
 
+type headerArgs []string
+
+func (h *headerArgs) String() string {
+	return fmt.Sprintf("%v", *h)
+}
+
+func (h *headerArgs) Set(value string) error {
+	*h = append(*h, value)
+	return nil
+}
```

Comment:
ditto

Reply:
We cannot use non-pointer `headerArgs` here because this function overwrites the pointed value.

Reply:
Oh you're right.